README.md

```diff
@@ -44,21 +46,94 @@ Check the provider documentation for provider-specific files and configuration t
 Start the deployment process
 
 ```
-$ docker run -ti -v (path to your directory):/root/data docker.io/osevg/openshifter create cluster01
+$ docker run -ti -v `pwd`:/root/data docker.io/osevg/openshifter create cluster01
```

Comment:
this assumes you run the command from the directly ... is it OK to assume?

//cc @jorgemoralespou @mhausenblas

Reply:
It should be stated explicitly

Reply:
So, you suggest we leave it as is?

Reply:
I'll leave as is and will add a note about using `pwd`